src/watchmaker/workers/salt.py

```
@@ -255,6 +260,22 @@ def _build_salt_formula(self, extract_dir):
                 to_directory=extract_dir
             )
 
+        bundled_content = os.sep.join(
+            (static.__path__[0], 'salt', 'content')
+        )
+        if os.path.exists(bundled_content):
```

This can't _not_ exist. That's an exception, let it raise.